docs/INTRODUCTION.md

```diff
@@ -1,63 +1,57 @@
 
-## Concepts
+[**OVE Services**](https://github.com/ove/ove-services) provides core functionality microservices within OVE such as the layout service. [**OVE Asset Services**](https://github.com/ove/ove-asset-services) provides an Asset Manager along with a collection of Asset Processing Services, used to upload archives or divide large networks or images into tiles, and serve these to the corresponding applications.
 
-OVE supports multiple ``spaces``. Each ``space`` is associated with one or more monitors, which may be attached to different computers, that together form a single display. This display may be a Large High Resolution Display (the [Data Observatory](https://www.imperial.ac.uk/data-science/data-observatory/) is 2.53m x 6.00m in size and 30,720 x 4,320 in resolution and consists of 64 monitors), but OVE is also suitable for use on much smaller displays.
+OVE also provides [graphical editors](https://github.com/ove/ove-editor) and [software development kits](https://github.com/ove/ove-sdks) that can be used to design and develop visualisation projects.
 
-In each space, OVE runs within a number of ``clients``. Each ``client`` is a browser window, typically full-screen on a single monitor, but a single ``client`` can span multiple windows if they are attached to the same computer. The arrangement of ``clients`` is described in the [``Spaces.json``](https://github.com/dsi-icl/ove/blob/master/packages/ove-core/src/client/Spaces.json) file.
+## Basic Concepts
 
-``Sections`` are rectangular regions within an ``oveCanvas``. Each ``section`` runs a single OVE ``app``. ``Sections`` may span multiple clients and can overlap.
+An OVE `server` installation supports multiple `spaces`. A `space` is a collection of monitors, which may be attached to different computers, that together form a single display. OVE is designed to be used in Large High Resolution Display environments (such as a [Data Observatory](https://www.imperial.ac.uk/data-science/data-observatory/) that is 2.53m x 6.00m in size and 30,720 x 4,320 in resolution, consisting of 64 monitors). But, OVE is also suitable for use on much smaller displays.
 
-An ``app`` displays something on the display. The official **core** ``apps`` each provide a way to display a distinct form of commonly used content.
-They include an ``HTML app`` that can be used to display any visualization that run as web apps (subject to the limitations described in the [list of pitfalls to avoid](./PITFALLS.md)). 
+In each space, OVE runs within a number of `clients`. An OVE `client` is a browser window and typically runs full-screen on a single `screen`. A `screen` can be a single monitor or can span multiple monitors if they were all attached to a single computer. The arrangement of `clients` is described in the [`Spaces.json`](https://github.com/dsi-icl/ove/blob/master/packages/ove-core/src/client/Spaces.json) file.
```

Comment:
```suggestion
In each space, OVE runs within a number of `clients`. An OVE `client` is a browser window and typically runs full-screen on a single `screen`. A `screen` can be a single monitor or can span multiple monitors that are attached to the same computer. The arrangement of `clients` is described in the [`Spaces.json`](https://github.com/dsi-icl/ove/blob/master/packages/ove-core/src/client/Spaces.json) file.
```

Reply:
Isn't it just two ways of saying the same thing? Is there any difference in meaning?

Reply:
- the tense of 'if they were' and 'they are' is different

- I think that 'the same computer' is clearer than 'a single computer', because there may be many computers, each with more than one monitor attached (whereas 'all attached to a single computer' could suggest that this only applies when the whole installation is controlled by a single computer)

Reply:
Not convinced on these points, James. I still believe that both versions would be interpreted by the reader in the same way, and as discussed offline, we can argue that either variant is more relevant in its own context. I would therefore stick to the original wording.